src/api/public/apidocs-new/paths/source_project_name_package_name_cmd_rebuild.yaml

```diff
@@ -0,0 +1,9 @@
+post:
+  deprecated: true
+  summary: Trigger a package rebuild.
+  description: |
+    Triggers a rebuild of the package. This endpoint
+    is broken and will be removed.
+    See [https://github.com/openSUSE/open-build-service/issues/13997](https://github.com/openSUSE/open-build-service/issues/13997).
```

Comment:
I don't think we need to point to github issues here. Instead you should tell people what to use instead 😸